src/api/supabase/docker-compose.yml

```diff
@@ -0,0 +1,158 @@
+# Usage
+#   Start:          docker-compose up
+#   With helpers:   docker-compose -f docker-compose.yml -f ./dev/docker-compose.dev.yml up
```

Comment:
I couldn't find the docs for this, Supabase doesn't have to most inclusive documentation.
But based on `./dev/docker-compose.dev.yml`, this will also run [inbucket](https://www.inbucket.org/)  for email testing and [meta](https://github.com/supabase/postgres-meta) for managing Postgres database like running queries and add roles.